devices.js

```diff
@@ -1409,6 +1433,18 @@ const devices = [
         toZigbee: [tz.tuya_cover_control, tz.tuya_cover_options],
         exposes: [e.cover_position()],
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0601', manufacturerName: '_TZE200_fctwhugx'},
+            {modelID: 'TS0601', manufacturerName: '_TZE200_g5wdnuow'},
```

Comment:
Please also identify the other devices through fingerprint instead of zigbeemodel

Reply:
finished, please review it.